docs/projects/docs-site/navbar/search-bar.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 1
+title: Search Bar in Navbar
+id: search-bar
+sidebar_label: Search Bar
+description: A detailed overview of adding Search bar in Navbar of TCET Open Source Website
+keywords: [tcet, open-source, software]
+---
+
+### Adding a search bar
+
+We are continuously working on improving our website to provide the best user experience. As part of this effort, we will be adding a **search bar functionality** in the near future. 
+
+This feature will make it easy for users to access the documentations and blogs on our website. Thank you for your patience and support as we strive to make our website more user-friendly and professional.
+
+> **Here are the steps we plan to take to implement this functionality:**
+
+1. Installing the `@docusaurus/plugin-search-local` package:
+
+    ```bash
+    npm install --save-dev @docusaurus/plugin-search-local
+    ```
+2. Adding the plugin to our `docusaurus.config.js` file:
+
+    ```js title="docusaurus.config.js" {3-15} showLineNumbers
+    module.exports = {
+        // ...
+        plugins: [
+            [
+                '@docusaurus/plugin-search-local',
+                {
+                    indexDocs: true, // Whether to index docs pages
+                    indexBlog: true, // Whether to index blog pages
+                    indexPages: true, // Whether to index static pages
+                    docsRouteBasePath: '/docs', // Base path for the docs route
+                    blogRouteBasePath: '/blog', // Base path for the blog route
+                    language: ['en'], // Supported languages
+                },
+            ],
+        ],
+        // ...
+        };
+    ```
+
+3. Adding the search bar to the navbar in our `themeConfig` file:
+
+    ```js title="docusaurus.config.js" {10-11} showLineNumbers
+    module.exports = {
+        // ...
+        themeConfig: {
+            // ...
+            navbar: {
+                // ...
+                items: [
+                    // ...
+                    {
+                        type: 'search',
+                        position: 'right',
+                    },
+                ],
+            },
+        },
+        // ...
+    };
+    ```
+
+4. Building our site:
+
+    ```bash
+    npm run build
+    ```
+
+5. Running our site:
+
+    ```bash
+    npm run start
+    ```
+
+
+
+## Congratulations 🎊
+
+Following these steps and points, we were able to create a beautiful and interactive navbar . It's final structure looks something like this:
+
+import heroImg from "./navbar-1.png";
+
+<img src={heroImg} />
```

Comment:
we do need to update this image later on when we actually add the search feature

Reply:
Yeah once we have that feature, I will be updating the image here.